change/@fluentui-react-positioning-448f1593-4e19-4760-806c-8532f539fdfb.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: export options `strategy`, `positionFixed`, `fallbackPositions` from `PositioningProps`. Deprecate `positionFixed`",
```

```suggestion
  "comment": "feat: export options `strategy`, `fallbackPositions` from `PositioningProps`. Deprecate `positionFixed`",
```

From a public API POV we are only exporting strategy, we can't remove `positionFixed` since it's public - but for react-positioning package

---

Updated change log :)